frontend/src/queries/schema.ts

```diff
@@ -1856,7 +1856,6 @@ interface WebAnalyticsQueryBase<R extends Record<string, any>> extends DataNode<
 
 export interface WebOverviewQuery extends WebAnalyticsQueryBase<WebOverviewQueryResponse> {
     kind: NodeKind.WebOverviewQuery
-    includeLCPScore?: boolean
```

Comment:
Weak opinion: but you could keep this, and have it link to web vitals. If we are going to remove it I'd wait until core web vitals is live, and disable it with the web vitals FF

Reply:
My problem with this value here is that it is not the same value as people will see in the Web Vitals UI because you're grabbing this from the session and not from the `$web_vitals` events like I'm doing. I'm afraid keeping this will make it more confusing than if we simply removed it.

As for keeping it and hiding it when people enable the flag: Yeah, I could do that if you felt strongly about it, but I also believe that we should not be displaying data that's not 100% true and/or doesn't promote any kind of insight so I'll keep this change for now